pkg/controller/migmigration/migrate.go

```diff
@@ -78,6 +79,9 @@ func (r *ReconcileMigMigration) migrate(migration *migapi.MigMigration) (time.Du
 	err = task.Run()
 	if err != nil {
 		log.Trace(err)
```

Comment:
Even through the logger will not trace _Conflict_, it would be more efficient to put the Log() after.